test/e2e/tests/ngJqSpec.js

```diff
@@ -0,0 +1,12 @@
+describe('Customizing the jqlite / jquery version', function() {
+
+  it('should be able to force jqlite', function() {
+    loadFixture("ngJq").andWaitForAngular();
+    expect(browser.executeScript('return window.angular.element !== window.jQuery')).toBe(true);
+  });
+
+  it('should be able to use a specific version jQuery', function() {
+    loadFixture("ngJqJquery").andWaitForAngular();
+    expect(browser.executeScript('return window.angular.element !== window.jQuery_2_0_0')).toBe(true);
```

Comment:
OK, so I see that you are indeed sending a block of script to be executed but I think it is still better not to try to run scripts from e2e tests